### PR TITLE
Include a check for letter_grade in passing_learners computation.

### PR DESCRIPTION
**Pull request by HassanJaveed84**
https://openedx.atlassian.net/browse/DESUPPORT-451

Analytics Pipeline Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] If you have a migration please contact data engineering team before merging.
  - [ ] Before merging run full acceptance tests suite and provide URL for the acceptance tests run.
  - [ ] A member of data engineering team has approved the pull request.

### CODE REVIEW

**Review by macdiesel — APPROVED** [review]

File: edx/analytics/tasks/insights/enrollments.py [hunk 1]
```diff
@@ -1881,13 +1881,14 @@ def insert_query(self):
         return """
         SELECT   all_enrollments.course_id AS course_id,
                  all_enrollments.mode AS mode,
-                 SUM(CASE WHEN closest_enrollment.passed_timestamp IS NOT NULL THEN 1 ELSE 0 END) AS passing_users
+                 SUM(CASE WHEN closest_enrollment.passed_timestamp IS NOT NULL AND closest_enrollment.letter_grade != '' THEN 1 ELSE 0 END) AS passing_users
```
Comment (pwnage101) [inline edx/analytics/tasks/insights/enrollments.py:1886]: If it was so difficult to find out the correct algorithm and implement it here, I would want comments added to the code or docstring to explain this scenario when the letter grade is and is not empty.

**Review by pwnage101 — APPROVED** [review]
Nit: add comments

File: edx/analytics/tasks/insights/enrollments.py (hunk 1 above)
Comment (HassanJaveed84) [reply edx/analytics/tasks/insights/enrollments.py:1886]: Added inline comments

**Review by bmedx — APPROVED** [review]

**Review by brianhw — APPROVED** [review]
LGTM.  👍

**Pull request merged**